Sources/WhisperKit/Core/Models.swift

```diff
@@ -171,6 +137,74 @@ public struct ModelComputeOptions {
     }
 }
 
+public struct ModelInfo: Identifiable, Hashable {
+    public let id = UUID()
```

Comment:
curious, why this property `public let id = UUID()` is needed? can it be uniquely identified by name?

Reply:
Needed to make it hashable, although I was using this for the picker at one point and may be vestigial, will check.